website/docs/advanced/override_grammar/basic.md

```diff
@@ -27,25 +27,25 @@ The rest are manipulating the config object.
 ## Grammar
 Hydra supports a rich [DSL](https://en.wikipedia.org/wiki/Domain-specific_language) in the command line.   
 Below are the parser rules from grammar.
-You can see the full grammar [here](https://github.com/facebookresearch/hydra/tree/master/hydra/grammar/Override.g4).
-
-```antlr4
-grammar Override;
+You can see the full grammar on GitHub
+([lexer](https://github.com/facebookresearch/hydra/tree/master/hydra/grammar/OverrideLexer.g4) and
```

Comment:
I think I didn't answer your question:

Yes, please add a paragraph about whitespace being supported in values and a mention that it has to be quoted for the shell.

Reply:
I added a sub-section on unquoted strings with shell examples in 6434fc6206726c6a51494e0607e0546c387eeeda, let me know if that looks good to you.